tests/php/bug49102-1.phpt

```diff
@@ -42,20 +42,20 @@ Content:
         <h2 class="title">Class synopsis</h2>
 
     <div class="classsynopsis">
-        <div class="ooclass"></div>
+        <div class="ooclass"><strong class="classname"></strong></div>
 
         <div class="classsynopsisinfo">
             <span class="ooclass">
-                <strong class="classname">SplStack</strong>
+                <span class="modifier">class</span> <strong class="classname"><strong class="classname">SplStack</strong></strong>
```

Comment:
@kocsismate seems #50 inserts a redundant strong tag (and possibly whitespace)

Reply:
I'll fix it asap